### PR TITLE
Prepare for part 3

### PR DESCRIPTION
**Pull request by twpayne**

### CODE REVIEW

File: count1_test.go [hunk 1]
```diff
@@ -19,15 +19,15 @@ var (
 )
 
 func BenchmarkCountBadIPs1(b *testing.B) {
-	badIPs, flows, _ := getFlowsAndBadIPs(benchNumFlows, benchNumBadIPs, benchHitRate)
+	badIPs, flows, _ := getBadIPsAndFlows(benchNumFlows, benchNumBadIPs, benchHitRate)
```
Comment (klauspost) [inline count1_test.go]: Argument order is wrong, it should be:
```suggestion
	badIPs, flows, _ := getBadIPsAndFlows(benchNumBadIPs, benchNumFlows, benchHitRate)
```

File: count1_test.go [hunk 2]
```diff
@@ -19,15 +19,15 @@ var (
...
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
 		result = CountBadIPs1(badIPs, flows)
 	}
 	b.StopTimer()
 }
 
-func getFlowsAndBadIPs(numBadIPs, numFlows int, hitRate float64) ([]net.IP, []*Flow, int) {
+func getBadIPsAndFlows(numBadIPs, numFlows int, hitRate float64) ([]net.IP, []*Flow, int) {
```
Comment (klauspost) [inline count1_test.go]: Arguments are ignored and the constants are used. That is why wrong param order doesn't affect things.

File: count1_test.go (hunk 1 above)
Comment (twpayne) [reply count1_test.go]: Thanks! I've just removed the arguments as they're not actually used (as you pointed out).

**Comment by klauspost** [comment]
👍🏼

**Pull request closed without merging**